### ISSUE TITLE
Math Libraries

### ISSUE THREAD

**Issue by ppenna** [nanvix/nanvix#213]
**Description**

The C math library ``libm`` port is missing on the following architectures, and we should fix that:
- [x] x86
- [ ] or1k

**Comment by ppenna** [comment]
@Theldus Could you give us more insights on this issue? It seems that @gbesnard find out that no patch was actually required.

**Comment by Theldus** [comment]
Yeah, looks like the only thing needed to do is append the libm inside the rules in src/lib/makefile and append in the LIBS var inside ubin/makefile and sbin/makefile.

**Comment by ppenna** [comment]
@gbesnard Could you then proceed with the pull request?
